tests/unit/orchestrators/test_output_utils.py

```diff
@@ -41,11 +39,8 @@ def test_output_artifact_preparation(create_step_run, local_stack):
         "output_name",
         str(step_run.id),
     )
+    output_artifact_uris["output_name"] = "/".join(
+        output_artifact_uris["output_name"].split("/")[:-1]
+    )
```

Comment:
is this what causes the test on Windows to fail, since it uses `\` instead of `/`?

Reply:
Yeah, updated this just now, thx.